MegaTermV1.0/assets/PYPKG!/MegaTerm.py

```diff
@@ -247,15 +247,15 @@ def print_file_content(filename):
             print(f"\033[37mDirectory '{directory}' not found.")
         except OSError:
             print(f"\033[37mFailed to remove directory '{directory}'.")
-    elif bash.startswith("rmfile "):
-        filename = bash[7:] 
-        try:
-            os.remove(filename)
-            #print(f"\033[37mFile '{filename}' removed.")
-        except FileNotFoundError:
-            print(f"\033[37mFile '{filename}' not found.")
-        except OSError:
-            print(f"\033[37mFailed to remove file '{filename}'.")
+    elif bash.startswith("./"):
+        file_path = bash[2:]
+        if os.path.exists(file_path) and os.path.isfile(file_path):
```

Comment:
# Improved security and error handling for user-provided file execution
haha...